test/metabase/embedding/settings_test.clj

```diff
@@ -199,11 +199,12 @@
 (deftest sync-origins-test
   ;; n.b. illegal combinations will be disallowed by [[embed.settings/check-and-sync-settings-on-startup!]], so we don't
   ;; test syncing for them.
-  (mt/with-premium-features #{:embedding :embedding-sdk}
+  (mt/with-premium-features #{:embedding :embedding-sdk :embedding-iframe-sdk}
```

Comment:
TODO: rename premium feature name to `embedding-simple` (to check with @albertoperdomo first because we need to update Harbormaster)